modules/@apostrophecms/page/ui/apos/components/AposPagesManager.vue

```diff
@@ -260,7 +260,7 @@ export default {
             busy: true,
             qs: {
               all: '1',
-              archived: this.relationshipField || this.pageSetMenuSelectionIsLive ? '0' : 'any',
+              archived: this.relationshipField || this.pageSetMenuSelectionIsLive ? false : 'any',
```

Comment:
It’s a query string, booleans aren’t a thing in query string parameter values.

That’s why the string representation is used in this particular case.

I recommend you solve this problem on the other end, by convincing batch operation `if` to accept strings `0` and `1` as false and true.